replica/database.cc

```diff
@@ -1995,6 +2026,11 @@ future<> database::do_apply(schema_ptr s, const frozen_mutation& m, tracing::tra
     auto uuid = m.column_family_id();
     auto& cf = find_column_family(uuid);
 
+    if (is_in_critical_disk_utilization_mode() && cf.is_eligible_to_write_rejection_on_critical_disk_utilization()) {
+        ++_stats->total_writes_rejected_due_to_out_of_space_prevention;
```

Comment:
Why not call `update_write_metrics_for_rejected_writes` here too?

Reply:
The function you mentioned updates the number of write attempts, write failures + failures due to out-of-space problems. This `if`-statement has been moved to `do_apply` and in this function we already update the remaining two metrics:
```
++_stats->total_writes;
// assume failure until proven otherwise
auto update_writes_failed = defer([&] { ++_stats->total_writes_failed; });
```